composer.json

```
@@ -25,11 +25,13 @@
     ],
     "require": {
         "php": "^8.0",
+        "http-interop/http-factory-guzzle": "^1.2.0",
         "sylius/paypal-plugin": "^1.5.2",
         "sylius/sylius": "~1.12.11",
         "symfony/dotenv": "^5.4 || ^6.4",
-        "symfony/runtime": "^5.4 || ^6.4",
-        "symfony/flex": "^2.4"
+        "symfony/flex": "^2.4",
+        "symfony/http-client": "^6.0.20",
```

Is it necessary as the requirement?
```suggestion
        "symfony/http-client": "^6.4",
```